Command line parameter not being escaped correctly, can't handle spaces in path

I was trying out the alpha package and hit a problem:

```
C:\Users\Brandon\.nuget\packages\minver\1.0.0-alpha.3\build\MinVer.targets(12,5): error MSB3073: The command "dotnet C:\Users\Brandon\.nuget\packages\minver\1.0.0-alpha.3\build\MinVer.Cli.dll C:\Users\Brandon\Documents\Visual Studio 2017\Projects\ConsoleApp141\ConsoleApp141 > "obj\Debug\netcoreapp2.1\min-ver-879a2.txt"" exited with code -532462766.
```

```
Unhandled Exception: System.Exception: Path 'C:\Users\Brandon\Documents\Visual' doesn't point at a valid workdir.
   at MinVer.Versioner.GetVersion(String path) in C:\projects\min-ver\MinVer\Versioner.cs:line 21
   at MinVer.Cli.Program.Main(String[] args) in C:\projects\min-ver\MinVer.Cli\Program.cs:line 14
```

Looks like the parameters being passed to the `dotnet` command in the targets file need to be updated to account for potential spaces in the paths.

--- a/MinVer.Cli/MinVer.targets
+++ b/MinVer.Cli/MinVer.targets
@@ -9,7 +9,7 @@
     <PropertyGroup>
       <OutputFile>$(IntermediateOutputPath)min-ver-$([System.Guid]::NewGuid().ToString().Substring(0, 5)).txt</OutputFile>
     </PropertyGroup>
-    <Exec Command="dotnet $(MSBuildThisFileDirectory)MinVer.Cli.dll $(MSBuildProjectDirectory) &gt; &quot;$(OutputFile)&quot;" />
+    <Exec Command="dotnet &quot;$(MSBuildThisFileDirectory)MinVer.Cli.dll&quot; &quot;$(MSBuildProjectDirectory)&quot; &gt; &quot;$(OutputFile)&quot;" />
     <PropertyGroup>
       <MinVerVersion>$([System.IO.File]::ReadAllText($(OutputFile)).Trim())</MinVerVersion>
     </PropertyGroup>

